lms/static/sass/views/_teams.scss

```diff
@@ -148,6 +149,200 @@
     @extend %t-weight4;
   }
 
+  .search-tools {
+    @extend %t-copy-sub1;
+    margin: ($baseline/10) $baseline;
+    color: $gray-l1;
+
+    p, span { // override
+      color: inherit;
+    }
+  }
+
+  .card {
+    @extend .clearfix;
+    position: relative;
+    display: inline-block;
+    width: 30%;
+    margin: ($baseline) 1.2%;
+    border: 1px solid $gray-l4;
+    border-bottom: 2px solid $gray-l3;
+    border-radius: 3px;
+    background-color: $gray-l5;
+    color: $black;
+
+    &:hover,
+    &:active,
+    &:focus {
+
+      .card-core-wrapper {
+        background-color: $white;
+      }
+
+      .card-meta-wrapper {
+        background-color: $gray-l5;
+      }
+    }
+
+    .card-core-wrapper {
+      @extend .clearfix;
+      transition: all .25s ease-in-out;
+      padding: ($baseline);
+    }
+
+    .card-title {
+      @extend %t-title5;
+      @extend %t-strong;
+      margin-bottom: .5em;
```

Comment:
Totally. I used the bourbon mixin because I wanted it to be tied to the size of the header. Thanks for pointing me to it!